cmd/hivechain/mod_txinvoke.go

```diff
@@ -151,6 +158,47 @@ func (m *modInvokeEmit) apply(ctx *genBlockContext) bool {
 			BlobHashes: sidecar.BlobHashes(),
 			Sidecar:    sidecar,
 		}
+	
+	case types.SetCodeTxType:
+			if !ctx.ChainConfig().IsPrague(ctx.Number(), ctx.Timestamp()) {
+				return false
+			}
+
+			auth := &types.Authorization{
+				ChainID: big.NewInt(1),
+				Address: recipient,
+				Nonce:   uint64(1),
+				V:       big.NewInt(1),
+				R:       big.NewInt(1),
+				S:       big.NewInt(1),
+			}
+			
+			signedAuth, err := types.SignAuth(auth, sender.key)
+		    if err != nil {
```

Comment:
might need to gofmt this

Reply:
Weird the formatting is correct in VS and Zed, but not in nvim.